dev/io.openliberty.io.opentelemetry.internal/src/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Originally, if a global OpenTelemetry instance is not set then GlobalOpenTelemetry.get()
+ * attempts to create and configure one automatically.
+ *
+ * To support per-application configuration, the global instance is set to a no-op
+ * implementation in this class
+ */
+package io.opentelemetry.api;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.api.trace.TracerBuilder;
+import io.opentelemetry.api.trace.TracerProvider;
+
+public final class GlobalOpenTelemetry {
+
+    private static final TraceComponent tc = Tr.register(GlobalOpenTelemetry.class);
+    private static boolean warned = false;
+
+    public static OpenTelemetry get() {
+        //Warns the user the first time they try to get the GlobalOpenTelemetry instance
+        //The GlobalOpenTelemetry instance should not be used as the entrypoint to telemetry functionality.
+        //OpenTCWMOT5000W: Cannot get GlobalOpenTelemetry.get
+        if (!warned) {
+            String msg = Tr.formatMessage(tc, "CWMOT5000.cannot.get.globalopentelemetry");
+            Tr.warning(tc, msg);
+            warned = true;
+        }
+        return OpenTelemetry.noop();
+    }
```

Comment:
I remembered that we use `GlobalOpenTelemetry.get()` to get the `OpenTelemetry` instance when using the agent, so we may see the Agent tests fail.

They may not fail, I'm not sure how the classloaders work when the agent is attached and which version of the class will be visible to our code. This version may be hidden completely by the agent in which case it would work.